constraint/buildutils/maketools.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+echo 'Starting tool build...'
+
+echo 'controller-gen'
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.1.8
```

Comment:
v0.2.2?

Reply:
Per the README in this directory, we have to be sure we are using an earlier version of controller-gen, as the newer version doesn't work with the CRD resource.